app/server/lib/ActiveDoc.ts

```diff
@@ -636,6 +648,7 @@ export class ActiveDoc extends EventEmitter {
     useExisting?: boolean,       // If set, document can be created as an overlay on
                                  // an existing sqlite file.
   }): Promise<ActiveDoc> {
+    console.log(`OPENOPENOPENOPENOPENOPENOPENOPEN ${this.docName} ANYTHING ELSE`);
```

Comment:
Stray debugging message?

Reply:
Yup, this and others. Thanks for catching this :sweat_smile: